ONSdigital/census-rm-toolbox
# Refactor loose python scripts into modules

# Motivation and Context
The base directory of this repo was getting a bit cluttered with loose python scripts, it seemed like a good time to start splitting off some modules

# What has changed
* Group some loose python scripts into modules

# How to test?
Build and deploy this to a GCP env, check the tool still work i.e. none of the aliases/script paths have broken.

# Links
https://trello.com/c/V3JRDv5E/977-reminders-batch-reminder-print-files-8
